WebHostLib/templates/siteMap.html

```diff
@@ -25,6 +25,7 @@ <h2>Base Pages</h2>
             <li><a href="/tutorial">Tutorials Page</a></li>
             <li><a href="/user-content">User Content</a></li>
             <li><a href="/weighted-settings">Weighted Settings Page</a></li>
+            <li><a href="{{url_for('stats')}}">Game Statistics</a></li>
```

Comment:
I'm happy you remembered this. Easy to forget but an important page nonetheless. Nicely done! 😄